src/CoreWCF.RabbitMQ.Client/src/CoreWCF.RabbitMQ.Client.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
+    <AssemblyName>CoreWCF.RabbitMQ.Client</AssemblyName>
```

Comment:
The client project should only declare .net6.0, otherwise it will generate a lib folder for each of the frameworks, include net472, which will result in it claiming to support net472. You will want to make the project references in the test project conditional otherwise when compiling the empty projects for .net framework, it will complain that this project only supports .NET 6.0.